wic/src/main/java/edu/asu/diging/wic/web/GetConceptWikiSummaryController.java

```diff
@@ -0,0 +1,39 @@
+package edu.asu.diging.wic.web;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.client.HttpClientErrorException;
+
+import edu.asu.diging.wic.core.conceptpower.IConceptpowerCache;
+import edu.asu.diging.wic.core.model.IConcept;
+import edu.asu.diging.wic.core.wiki.IWikiConnector;
+
+@Controller
+@PropertySource(value = "classpath:/config.properties")
+public class GetConceptWikiSummaryController {
+
+    @Autowired
+    private IConceptpowerCache cache;
+
+    @Autowired
+    private IWikiConnector wikiConnector;
+
+    @GetMapping("/concept/{conceptId}/wiki-summary")
+    public ResponseEntity<String> getConceptSummary(@PathVariable("conceptId") String conceptId) {
+        IConcept concept = cache.getConceptById(conceptId);
+        if (concept == null) {
+            return new ResponseEntity<>("", HttpStatus.NOT_FOUND);
+        }
+        try {
+            return new ResponseEntity<>(wikiConnector.getSummary(wikiConnector.getPageTitle(concept)), HttpStatus.OK);
+        } catch (HttpClientErrorException ex) {
+            return new ResponseEntity<>("", HttpStatus.NO_CONTENT);
```

Comment:
that seems like the wrong code, the request has not succeeded has it? or when is this exception being thrown?
> The HTTP 204 No Content success status response code indicates that a request has succeeded, but that the client doesn't need to navigate away from its current page.